webbpsf/webbpsf_core.py

```diff
@@ -1564,6 +1720,10 @@ def show_notebook_interface(instrument):
     from IPython.display import display, clear_output
     from matplotlib import pyplot as plt
 
+
+    if isinstance(instrument, string_types):
```

Comment:
I noticed a typo here ('instrumet') but also I'm :-1: on [stringly typed](http://c2.com/cgi/wiki?StringlyTyped) interfaces. If there's a need for a convenience helper function, we should make it a wrapper.